cli.ts

```diff
@@ -66,6 +68,23 @@ async function cli(args: ParsedArgs) {
     validateStatus: () => true,
   })
 
```

Comment:
I think we should also log the response here so they can see the initial return if they'd like

Reply:
also the original status